packages/elastic-apm-synthtrace/src/scripts/examples/01_simple_trace.ts

```diff
@@ -38,8 +39,16 @@ const scenario: Scenario = async (runOptions: RunOptions) => {
             .duration(1000)
             .success()
             .children(
+              instance
+                .span('GET /products', 'sql', 'mysql')
+                .dbStatement('SELECT * FROM PRODUCTS')
```

Comment:
there's also a `fields` method (I think) that you should be able to use. I'd like to keep top-level methods for often-used fields or things that are not a 1:1 mapping w/ fields